dspace/config/registries/dublin-core-types.xml

```diff
@@ -268,6 +268,12 @@
     <qualifier>uri</qualifier>
     <scope_note>Uniform Resource Identifier</scope_note>
   </dc-type>
+  
+
+
+
+  
+
```

Comment:
Can we revert all the changes to `dublin-core-types.xml`?  It looks like this PR is now just adding spaces / empty lines.